plugin/backend_test.go

```diff
@@ -17,6 +18,12 @@ const (
 	googleCredentialsEnv = "GOOGLE_TEST_CREDENTIALS"
 )
 
+func skipIfAccTestNotEnabled(t *testing.T) {
+	if _, ok := os.LookupEnv("ACC_TEST_ENABLED"); !ok {
+		t.Skip(fmt.Printf("Skipping accpetance test %s; ACC_TEST_ENABLED is not set.", t.Name()))
```

Comment:
nit: This results in
```Skipping accpetance test TestLogin_IAM; ACC_TEST_ENABLED is not set.    backend_test.go:23: 68 <nil>```

I think if we use fmt.Sprintf we can avoid the awkward nil output.

```suggestion
		t.Skip(fmt.Sprintf("Skipping accpetance test %s; ACC_TEST_ENABLED is not set.", t.Name()))
```

Reply:
Nice catch this is definitely better.